library/index.c

```diff
@@ -323,24 +323,25 @@ int eblob_index_blocks_fill(struct eblob_base_ctl *bctl)
 }
 
 
-static struct eblob_disk_control *eblob_find_on_disk(struct eblob_backend *b,
-		struct eblob_base_ctl *bctl, struct eblob_disk_control *dc,
+static int eblob_find_on_disk(struct eblob_backend *b,
```

Comment:
Logically these loops does same thing as loops in previous implementation without pread.
Loops in this PR reads by blocks items, but can be simplified to reading items one by one. Reading items one by one may be less efficient.
Should I add more comments in this loops or should I rewrite it to less efficient algorithms (reading one by one)?